label_sync/labels.yaml

```diff
@@ -706,6 +710,11 @@ repos:
         name: area/hyperkube
         target: both
         addedBy: label
+      - color: 0052cc
+        description: Issues or PRs related to the Release Engineering subproject
+        name: area/releng
```

Comment:
There is already an `area/release-infra` label in k/k today. Can you add the `previously` field here so that issues with that label can be automatically migrated to this one?

https://github.com/kubernetes/test-infra/blob/74f8a31c55613d1732a195ef2641fa109b477fcb/label_sync/labels.yaml#L5

Reply:
@idealhack -- `RelEng` is a fairly common compound for `Release Engineering` and it makes it easier to label issues, but I'm happy to change it if people feel strongly about having it be more descriptive. cc: @BenTheElder @fejta 

@nikhita -- Good catch. I'll update once we bikeshed on the name, since I'll have to update [here](https://github.com/kubernetes/sig-release/pull/453) as well.

/hold

Reply:
For sure! Let's see what everyone thinks. :)

Reply:
SIG-bikeshed is not strong in me today 😉 - RelEng is a common term around Google but either seems reasonable.

Reply:
I agree with @idealhack `release-eng` would be better, it parallels `release-team`

No catchall `area/release` plz, there's a reason SIG release wants these as two distinct subprojects.

Reply:
sig/release is effectively the catchall